### DISCUSSION TITLE
I can't install DeZog 2.4.3

### DISCUSSION THREAD

**Discussion by jjmarin** [maziac/DeZog#77] [General]
Hi, I'm new with DeZog, and I've found that I can't install DeZog 2.4.3 on my system. This is  a Fedora 35 x86 64bits with Visual Studio Code version 1.49.1 from the official rpm repo

Version: 1.49.1
Commit: 58bb7b2331731bf72587010e943852e13e6fd3cf
Date: 2020-09-16T23:23:33.049Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.15.14-200.fc35.x86_64

When I search the DeZog extension, I found the 2.4.3 
![Screenshot from 2022-01-22 19-57-02](https://user-images.githubusercontent.com/224784/150652226-4bea62d6-b1e6-45ad-aa9a-b4277f6b6ea8.png)

However, after installing it, it turns into the 1.5.4!
![Screenshot from 2022-01-22 19-57-39](https://user-images.githubusercontent.com/224784/150652255-030798cf-e3df-4ddd-a480-d089bfe631a3.png)

If I try to find another versions, but then the version 1.5.4 seems to be the lastest available.
![Screenshot from 2022-01-22 19-58-26](https://user-images.githubusercontent.com/224784/150652285-92d070e2-dc93-4713-8df0-a8e19206728e.png)

When I get back again to the extensions menu, everything says I installed version 1.5.4, not version 2.4.3
![Screenshot from 2022-01-22 19-59-17](https://user-images.githubusercontent.com/224784/150652406-d9b1ff45-ebb6-46ec-b2d0-7ce21d9a2c01.png)

Whats the problem here? What I'm doing wrong?
Thank you in advance

**Comment by maziac** [comment]
Have you tried to install the vsix directly from here:
https://github.com/maziac/DeZog/releases/tag/v2.4.3

**Reply by jjmarin** [reply]
Hi, I've just tried and I've got the following error:
![Screenshot from 2022-01-22 21-39-43](https://user-images.githubusercontent.com/224784/150654698-668b468d-2089-4840-a32a-e7ce3da76972.png)

**Reply by maziac** [reply]
Yes, you need to update vscode to almost newest version. I think 1.62 or 1.63.

**Comment by jjmarin** [comment]
Thank you. I've finally installed after update to version 1.63.2

**Comment by maziac** [comment]
No problem.
